{{ cookiecutter.project_name_dashed }}/.tool-versions

```diff
@@ -0,0 +1,2 @@
+poetry 1.8.5
```

Comment:
As discussed before, please prompt the user for the Poetry version as well. Otherwise we either have to change this every few weeks/months _or_ have to remember to update this manually every time we start a project. Yes, it's one additional prompt, but considering how much we get from this template, relative to how rarely we create new projects, I think it's absolutely acceptable to spend 10 minutes going through the prompts.